### PR TITLE
feat: added code to trigger the QR Code Scan step on ICP

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

To trigger the QR Code scan step, we created specific events `icQRCodeScan` and `icQRCodeBack`.

# Changes

- Create event `icQRCodeScan` that is dispatched by the QR Code Button.
- Create event `icQRCodeBack` that is dispatched by the Cancel button in the QR Code Scan step.
- Assign them to navigate to the appropriate Wizard Step, using new function `goToWizardStep`. 

# Test
Manual testing done on local environment.

### CODE REVIEW

File: src/frontend/src/icp/components/send/IcSendModal.svelte [hunk 1]
```diff
@@ -177,6 +179,11 @@
 	setContext<EthereumFeeContextType>(ETHEREUM_FEE_CONTEXT_KEY, {
 		store: initEthereumFeeStore()
 	});
+
+	const goToWizardStep = (stepName: WizardStepsSend) => {
```
Comment (peterpeterparker) [inline src/frontend/src/icp/components/send/IcSendModal.svelte:183]: Should we extract this in a utils to avoid duplicated code?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/icp/components/send/IcSendModal.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/components/send/IcSendModal.svelte:183]: Yes, agreed! I was going to do it in the next PR eheh

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/components/send/IcSendModal.svelte:183]: Changing it with PR #1378

**Pull request merged**